app/lib/frontend/backend.dart

```diff
@@ -265,12 +265,19 @@ class Backend {
     await uiPackageCache.invalidateUIPackagePage(invite.packageName);
   }
 
-  /// Removes obsolete (== expired more than a day ago) invites from Datastore.
+  /// Removes obsolete/expired invites from Datastore.
   Future deleteObsoleteInvites() async {
     final query = db.query<models.PackageInvite>()
-      ..filter('expires <', DateTime.now().subtract(Duration(days: 1)));
+      ..filter('expires <', DateTime.now().toUtc());
```

Comment:
The reason  to wait a day is to avoid clock drift issues...

But our servers shouldn't drift by hours :)

Reply:
so I doubt it's important :)

Reply:
Let's deal with that once we have complaints about it :)